### PR TITLE
speech: update to v1

### PR DESCRIPTION
**Pull request by stephenplusplus**
Update the Speech API to v1.

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Note sure: #1930

**Comment by lukesneeringer** [comment]
For now, my vote is to leave them included (they may or may not _actually run_). We are working on a longer-term testing strategy for these.

It is worth noting that while the _development_ does not happen here, this is more or less the canonical location for GAPICs. :-/

I think the best way to do that ultimately is to have the generation process make a PR to this repo, and have the tests run that way, and rely less on api-client-staging as a canonical location.

**Comment by stephenplusplus** [comment]
That's cool.

File: packages/speech/src/index.js [hunk 1]
```diff
@@ -492,6 +432,12 @@ Speech.prototype.createRecognizeStream = function(config) {
     throw new Error('A recognize request requires a configuration object.');
   }
 
+  config = extend(true, {
+    config: {
+      languageCode: 'en-US'
```
Comment (lukesneeringer) [inline packages/speech/src/index.js]: This is bad. The client library needs to not assume `en-US`. The user must provide this explicitly. (The backend API changed this for a reason.)

**Pull request merged**